CoreWhatsNewView/src/au/gov/asd/tac/constellation/views/whatsnew/whatsnew.txt

```diff
@@ -1,6 +1,10 @@
 == 3030-12-31 Getting Started
 <p>If you're new to Constellation, read the <a href="" helpId="au.gov.asd.tac.constellation.functionality.gettingstarted">getting started guide</a>.</p>
 
+== 2022-10-19 Export from Analytic View
+<p>The ability to export the results from running an analytic in the Analytic View to a CSV or Excel file have been added.</p>
```

Comment:
I think you could omit "to a CSV or Excel file" from this line so it flows a little nicer. Would also need to change "have" to "has"